Modules/@babylonjs/react-native-iosandroid/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@babylonjs/react-native-iosandroid",
+  "title": "React Native Babylon for iOS and Android",
+  "version": "0.0.1",
+  "description": "Babylon Native integration into React Native",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/babylonjs/BabylonReactNative.git",
+    "baseUrl": "https://github.com/babylonjs/BabylonReactNative"
+  },
+  "keywords": [
+    "react-native"
+  ],
+  "author": {
+    "name": "Your Name",
+    "email": "yourname@email.com"
+  },
+  "homepage": "https://github.com/BabylonJS/BabylonReactNative#readme",
+  "license": "MIT",
+  "licenseFilename": "LICENSE",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "base-64": "^0.1.0",
+    "semver": "^7.3.2"
+  },
+  "peerDependencies": {
+    "@babylonjs/core": "^5.0.0-rc.4",
+    "@babylonjs/react-native": "version",
+    "react": ">=16.13.1",
+    "react-native": ">=0.63.1",
+    "react-native-permissions": "^3.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.8.4",
+    "@babel/runtime": "^7.8.4",
+    "@babylonjs/core": "5.0.0-rc.4",
+    "@rnw-scripts/eslint-config": "0.1.6",
+    "@rnw-scripts/ts-config": "0.1.0",
+    "@types/base-64": "^0.1.3",
+    "@types/jest": "^25.2.1",
+    "@types/react": "^16.9.32",
+    "@types/react-native": "^0.63.1",
```

Comment:
I think we can remove this since we shouldn't have any TypeScript in this module?